src/core/constants/species.ts

```diff
@@ -1,6 +1,6 @@
 import { msg } from "@lingui/macro"
 
-const species = {
+const species = /* @__PURE__ */ (() => ({
```

Comment:
This constant could not be tree-shaked because of `msg` calls.
```ts
const species = {
  Cardano: [
    {
      startsAt: 0,
      name: msg`Ghost`, // msg will make it appear in the PO
      icon: "Ghost",
    },
```

Bundler simply don't know would this call produce side effects or not. So this code considered not safe to be deleted. 

`/* @__PURE__ */` could be applied only for `new` or `CallExpression` statemetns, so to achieve tree-shaking here code have to be wrapped in IIFE and annotated.